launch_xml/test/launch_xml/executable.xml

```diff
@@ -0,0 +1,5 @@
+<launch>
+    <executable cmd="ls" cwd="/" name="my_ls" args="-l -a -s" shell="true" output='log' launch-prefix='$(env LAUNCH_PREFIX)'>
```

Comment:
@ivanpauno nit: I wonder if just collapsing `args` into `cmd` wouldn't be simpler: `ls -l -a -s`